app/components/Receive/ReceiveExplanation/styles.scss

```diff
@@ -0,0 +1,28 @@
+.receiveExplanation {
+  width: 250px;
```

Comment:
This doesnt seem to be wide enough...  the container in sketch is 300px can we use that instead